src/e2e/puppeteer-environment.ts

```diff
@@ -9,28 +8,23 @@ const PuppeteerEnvironment: Environment = {
   async setup(global, options) {
     builtinEnvironments['happy-dom'].setup(global, options)
 
+    const args = [
+      '--deterministic-fetch',
+      '--disable-dev-shm-usage',
+      '--disable-features=IsolateOrigins',
+      '--disable-setuid-sandbox',
+      '--disable-site-isolation-trials',
+      '--no-first-run',
+      '--no-sandbox',
+      '--no-zygote',
+    ]
+
     const browser = await puppeteer
-      .launch({
-        headless: true,
-        // Disable Chrome features that crash GitHub Actions with "Protocol error (Target.createTarget): Target closed."
-        // See: https://stackoverflow.com/a/66994528/480608
-        // List of Chromium switches: https://peter.sh/experiments/chromium-command-line-switches/
-        args: [
-          '--deterministic-fetch',
-          '--disable-dev-shm-usage',
-          '--disable-features=IsolateOrigins',
-          '--disable-setuid-sandbox',
-          '--disable-site-isolation-trials',
-          '--no-first-run',
-          '--no-sandbox',
-          '--no-zygote',
-        ],
-        // Increase timeout to avoid: "TimeoutError: Timed out after 30000 ms while trying to connect to the browser! Only Chrome at revision r869685 is guaranteed to work."
-        timeout: 60000,
-      })
+      .connect({ browserWSEndpoint: `ws://localhost:7566?${args.join('&')}` })
       // catch and log a launch error, otherwise it will not appear in the CI logs
       .catch((err: Error) => {
-        console.error('Error launching puppeteer:', err)
+        // eslint-disable-next-line no-console
+        console.log('Could not connect to browserless.\nMake sure to run `yarn browserless` before running the tests.')
```

Comment:
Use `console.error` so you don't have to suppress the linter.

Reply:
`console.error` and `console.warn` were swallowed by vitest and were very easy to miss in the console, hence falling back to `console.log` here.

Reply:
What about `console.info`?

Reply:
Also not, I'm afraid. Leaving the `console.log` in case something went wrong with browserless, but will handle automatic setup and teardown in a script as mentioned.